setup.py

```diff
@@ -36,6 +36,7 @@
     "diffusers >= 0.31.0, < 0.32.0",
     "huggingface_hub >= 0.24.7",
     "sentence-transformers == 3.2.1",
+    "wget",
```

Comment:
We should not use that to download files on the HF hub but: https://huggingface.co/docs/huggingface_hub/v0.26.3/en/package_reference/file_download#huggingface_hub.hf_hub_download

Reply:
This one is already removed.